MAINTAINERSHIP.md

```diff
@@ -0,0 +1,42 @@
+Maintainers agree to operate under this set of guidelines:
+
+#### Authority
+
+Maintainers are trusted to close issues, merge pull requests, and publish crates to cargo.
+
+#### Categories of Work
+
+0. Minor
+    * updating the changelog
+    * requires no approval
+1. Normal
+    * librocksdb-sys updates
+    * API tracking code in the rocksdb crate that does not change control flow
+    * breaking changes due to removed functionality in rocksdb
+    * require 1 approval from another maintainer. if no maintainer is able to be reached for 2 weeks, then progress may be made anyway
+    * patch (and post-1.0, minor) releases to crates.io that contain only the above work
+2. Major
+    * breaking API changes that are not direct consequences of underlying rocksdb changes
+    * refactoring, which should generally only be done for clearly functional reasons like to aid in the completion of a specific task
+    * require consensus among all maintainers unless 2 weeks have gone by without full participation
+    * if 2 weeks have gone by after seeking feedback, and at least one other maintainer has participated, and all participating maintainers are in agreement, then progress may be made anyway
+    * if action is absolutely urgent, an organization owner may act as a tiebreaker if specifically requested to do so and they agree that making a controversial decision is worth the risk. This should hopefully never occur.
+  
+If any maintainer thinks an issue is major, it is major.
+
+#### Changelog Maintenance
+
+* If you are the one who merges a PR that includes an externally-visible change, please describe the change in the changelog and merge it in.
+
+#### Releasing, Publishing
```

Comment:
Should a bullet be added saying this project follows [semver rules?](https://semver.org/)

Reply:
added